pyquil/paulis.py

```diff
@@ -22,7 +22,8 @@
 import numpy as np
 import copy
 
-from typing import Dict, FrozenSet, Iterable, Iterator, List, Optional, Tuple, Union
+from typing import Callable, Dict, FrozenSet, Iterable, Iterator, List, Optional, Tuple, Union
+from typing import Sequence as SequenceT
```

Comment:
The only use of the existing `collections.abc.Sequence` appears to be an `isinstance` check. `typing.Sequence` inherits from `abc.Sequence`, and so also works for `isinstance` checks. As such, I would just get rid of the `collections.abc.Sequence` import, and change this to `from typing import Sequence`